src/core/pointcloud/qgspointclouddataprovider.cpp

```diff
@@ -176,3 +181,94 @@ QVariant QgsPointCloudDataProvider::metadataClassStatistic( const QString &, con
 {
   return QVariant();
 }
+
+QVector<QMap<QString, QVariant>> QgsPointCloudDataProvider::identify(
+                                QgsPointCloudLayer *layer,
+                                float maximumError,
```

Comment:
A cleaner API would be to remove rootErrorPixels, and require that maximumError is specified in the data provider's native units. (I.e. require that the caller does the work to convert their desired maximum error into the correct units.) That would allow removal of rootErrorPixels, because it can be calculated in this method directly from the index root node.